test/Index/index_swift_only_systemmodule.swift

```diff
@@ -99,7 +99,7 @@ print(someFunc())
 // RUN:     -index-system-modules \
 // RUN:     -index-ignore-stdlib \
 // RUN:     -index-store-path %t/idx \
-// RUN:     -sdk %t/SDK \
+// RUN:     -sdk %t%{fs-sep}SDK \
```

Comment:
Are these needed after you changed the path comparison?

Reply:
Right, they shouldn't. I dropped that change.